packages/igx-templates/igx-ts-legacy/projects/_base/files/src/index.html

```diff
@@ -9,7 +9,6 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,700" rel="stylesheet">
```

Comment:
@viktorkombov only this change should be left. The other changes I guess can come from an old version of the branch. Do you want us to change the client part of the CodeGen, or you have a plan to do it?